src/explore-education-statistics-common/src/modules/table-tool/components/__tests__/LocationFiltersForm.test.tsx

```diff
@@ -628,4 +660,52 @@ describe('LocationFiltersForm', () => {
       expect(handleSubmit).toHaveBeenCalledWith(expected);
     });
   });
+
+  test('sets the step heading to `Choose locations` if multiple location types', () => {
+    render(
+      <LocationFiltersForm
+        {...testWizardStepProps}
+        options={testLocationsFlat}
+        onSubmit={noop}
+      />,
+    );
+
+    expect(
+      screen.getByRole('heading', {
+        name: 'Step 1 (current) Choose locations',
+      }),
+    ).toBeInTheDocument();
+  });
+
+  test('sets the step heading based on location type if only one type', () => {
+    render(
+      <LocationFiltersForm
+        {...testWizardStepProps}
+        options={testSingleLocationType}
+        onSubmit={noop}
+      />,
+    );
+
+    expect(
+      screen.getByRole('heading', {
+        name: 'Step 1 (current) Choose Countries',
+      }),
+    ).toBeInTheDocument();
+  });
+
+  test('sets the step heading to `Choose locations` if single location type is not in the locationLevelsMap', () => {
+    render(
+      <LocationFiltersForm
+        {...testWizardStepProps}
+        options={testSingleLocationTypeUnknown}
```

Comment:
Same as above - would inline the test variable

Reply:
Done